lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/TrashBinFallback.java

```diff
@@ -0,0 +1,40 @@
+package org.enso.desktopenvironment;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+
+public class TrashBinFallback {
```

Comment:
This class shouldn't be `public`.
```suggestion
class TrashBinFallback {
```